src/components/UserProfile/UserProfileContent.js

```diff
@@ -12,36 +13,41 @@ export default function UserProfileContent({ userDataInfo }) {
     street,
     zip,
     city,
+    url,
   } = userDataInfo
 
   return (
-    <ProfileWrapper>
-      <NameStyled>{firstName + ' ' + lastName}</NameStyled>
-      <CompanyStyled>{company}</CompanyStyled>
-      <DetailListStyled>
-        <DetailItemHeading>Kontaktdaten</DetailItemHeading>
-        <DetailItemContainer>
-          <DetailItemContent>T: {phone}</DetailItemContent>
-          <DetailItemContent>M: {mail}</DetailItemContent>
-        </DetailItemContainer>
-        <DetailItemHeading>Adresse</DetailItemHeading>
-        <DetailItemContainer>
-          <DetailItemContent>{street}</DetailItemContent>
-          <DetailItemContent>{zip + ' ' + city}</DetailItemContent>
-        </DetailItemContainer>
-      </DetailListStyled>
-      <Link to="/edit-user">
-        <EditButton to="/edit-user">Bearbeiten</EditButton>
+    <>
+      <ShareUserProfile userDataInfo={userDataInfo} />
+      <ProfileWrapper>
+        <NameStyled>{firstName + ' ' + lastName}</NameStyled>
+        <CompanyStyled>{company}</CompanyStyled>
+        <DetailListStyled>
+          <DetailItemHeading>Kontaktdaten</DetailItemHeading>
+          <DetailItemContainer>
+            <DetailItemContent>T: {phone}</DetailItemContent>
+            <DetailItemContent>M: {mail}</DetailItemContent>
+            <DetailItemContent>W: {url} </DetailItemContent>
+          </DetailItemContainer>
+          <DetailItemHeading>Adresse</DetailItemHeading>
+          <DetailItemContainer>
+            <DetailItemContent>{street}</DetailItemContent>
+            <DetailItemContent>{zip + ' ' + city}</DetailItemContent>
+          </DetailItemContainer>
+        </DetailListStyled>
+        <Link to="/edit-user">
+          <EditButton to="/edit-user">Bearbeiten</EditButton>
         </Link>
-    </ProfileWrapper>
+      </ProfileWrapper>
+    </>
   )
 }
 
```

Comment:
The names of your styled components are really long, which hidners readability, imho. Make them short and sweet.